lib/parser/interface-list-parser.ts

```diff
@@ -31,77 +31,77 @@ export class InterfaceListParser extends ParserBase {
       this.debug('parse i ' + this.pos.i);
 
       this.advanceWhitespace();
-
-      const port = generics ?
-        new OGeneric(this.parent, this.getToken().range.copyExtendEndOfLine()) :
-        new OPort(this.parent, this.getToken().range.copyExtendEndOfLine());
-
       if (this.getToken().text === ')') {
         this.consumeToken();
         break;
       }
-
       const nextWord = this.getToken();
-      if (nextWord.getLText() === 'type') {
-        this.consumeToken();
-        port.lexerToken = this.consumeToken();
-        ports.push(port);
+
+      if (nextWord.getLText() === 'package') {
```

Comment:
```suggestion
      if (generics && nextWord.getLText() === 'package') {
```
port lists should not contain packages, similar for types